### ISSUE TITLE
Add item.bodyDisplayString to API documentation ?

### ISSUE THREAD

**Issue by RobTrew** [hbsw/tp-issues#229]
item.bodyDisplayString is a very useful resource for scripting TaskPaper 3 – perhaps worth adding to the list of documented item properties in http://guide.taskpaper.com/Item.html ?

**Comment by jessegrosjean** [comment]
I'll do this for the next release, but the reason I've avoided it is I'm still not quite sure about the name or exact definition of what it does. Right now I use it to ...

When I want to show an item in a menu (or other UI) without a "mess of distracting syntax". So right now I strip off leading dashes, trailing :'s and trailing tags. Why now interior tags... not sure! :)

In the future if I add support for **bold** or _italic_ would this method strip out those *_'s?

I'm not sure all these things need answer before making it public. I'm just not sure about the answers myself and wonder what you think as another user of the API.

**Comment by RobTrew** [comment]
Makes sense. 

My initial vote would be for simple prefix and tag stripping – leaving any informal inline MD paraphernalia in place.

(It's useful for the API to be able to see them, for example if passing them straight through to MarkedApp2 for CSS formatted printing)

**Comment by RobTrew** [comment]
I wonder if there's an argument for a method which has N boolean arguments to specify which of the modal layers are to be removed, and which included ?

e.g. item.bodyDisplayString(blnWithTags, blnWithLineTypeSyntax, blnWithFormattingAsterisks) ?

**Comment by jessegrosjean** [comment]
Sorry, just to make things difficult I've decided to rename `bodyDisplayString` to `bodyContentString`. I think that better describes the purpose, and I used a similar name for a similar purpose in FoldingText. Plus TaskPaper uses "display" in the API mostly when talking about items visible onto screen, which is unrelated.

I still don't have a good answer for how internal formatting and tags should be handled, but I'm going to leave the current behavior.
